frontend/src/global/constants.ts

```diff
@@ -64,7 +64,8 @@ export const ROUTES = {
     THEME_NAVIGATION: '/console/theming/navigation',
     THEME_FOOTER: '/console/theming/footer',
     INSTANCE_SETTINGS_GENERAL: '/console/settings/general',
-    HERO: '/console/theming/hero'
+    HERO: '/console/theming/hero',
+    COURSES: '/console/courses/manage'
```

Comment:
Nit: add trailing comma.
```suggestion
    COURSES: '/console/courses/manage',
```

Reply:
There's still no trailing comma. What this something that the linter complained about?

Reply:
That's right. lint-fix also removes them.